app/models/user.rb

```diff
@@ -4,4 +4,21 @@ class User < ActiveRecord::Base
   validates :credit_card_form_url, url: {allow_blank: true, no_local: true, schemes: ['http', 'https']}
   validates :duns_number, duns_number: true
   validates :email, allow_blank: true, email: true
+
+  def from_oauth_hash(auth_hash)
+    set_if_blank('name', auth_hash)
+    set_if_blank('email', auth_hash)
+    self.github_login = auth_hash[:info][:nickname]
```

Comment:
hahah never! I left this comment bc I found it to be super confusing when writing the code. oauth hash has a bunch of fields with `name` in the them and it took be a bit to figure out which one was `github_login` -- octokit response used `github_login` key but omniauth does not return that key!